docs2/site/docs/migrations/migration5.md

```diff
@@ -196,8 +202,44 @@ If you use provided extension methods upon `IGraphQLBuilder` then your code does
 
 ### 13. Changes caused by GraphQL-Parser v8
 
+- The `GraphQL.Language.AST` namespace and all classes from it have been removed in favor of ones
+  from `GraphQLParser.AST` namespace in GraphQL-Parser project. Examples of changed usages:
+  - `GraphQL.Language.AST.Document` -> `GraphQLParser.AST.GraphQLDocument`
+  - `GraphQL.Language.AST.IValue` -> `GraphQLParser.AST.GraphQLValue`
+  - `GraphQL.Language.AST.Field` -> `GraphQLParser.AST.GraphQLField`
+  - `GraphQL.Language.AST.SelectionSet` -> `GraphQLParser.AST.GraphQLSelectionSet`
+  - `GraphQL.Language.AST.IHaveDirectives` -> `GraphQLParser.AST.IHasDirectivesNode`
+  - `GraphQL.Language.AST.IType` -> `GraphQLParser.AST.GraphQLType`
+- Some APIs become to work with `GraphQLParser.ROM` struct instead of `string`:
+  - `ExecutionResult.Query`
+  - `Metrics.SetOperationName`
+  - `IComplexGraphType.GetField`
+  - `QueryArguments.Find`
+  - `SchemaDirectives.Find`
+  - `SchemaDirectives.this[]`
+  - `SchemaDirectives.Dictionary`
+  - `ValidationContext.GetFragment`
+  - All `ValidationError`'s constructors take _originalQuery_ as `ROM`
 - `OperationType` and `DirectiveLocation` enums were removed, use enums from `GraphQLParser.AST` namespace
 - `SourceLocation` struct was removed, use `GraphQLLocation` from `GraphQLParser.AST` namespace
+- `ErrorLocation` struct was removed, use `Location` from `GraphQLParser` namespace
+- `IResolveFieldContext.SubFields` and `IExecutionStrategy.GetSubFields` returns dictionary with
+   values of tuple of queried field and its field definition
+- All scalars works with `GraphQLParser.AST.GraphQLValue` instead of `GraphQL.Language.AST.IValue`
+- `IInputObjectGraphType.ToAST` returns `GraphQLParser.AST.GraphQLObjectValue` instead of `GraphQL.Language.AST.IValue`
```

Comment:
Should add in here that the `CoreToVanillaConverter` was removed

Reply:
`CoreToVanillaConverter` was also in `GraphQL.Language.AST`

Reply:
OK, `GraphQL.Language`